js/viz/series/base_series.js

```diff
@@ -485,7 +485,35 @@ Series.prototype = {
         }
     },
 
-    _drawElements: function(animationEnabled, firstDrawing, translateAllPoints) {
+    _preparePositionSegments() {
+        const that = this;
```

Comment:
Why we use `that`? `that` equal `this` here or not?

Reply:
In this case, not really,
specifically in the charts, historically, in the base series that denotes the current series that we are working with, in this case I need to use it inside the loop

If this is changed, then immediately in the entire file.